### PR TITLE
Feature/02 form functionality

### PR DESCRIPTION
**Pull request by annnuuuh**
What (if any) features are you implementing?
-Form functionality in JS & timer display in html
What (if anything) did you refactor?
-Cleaned up my JS in a second commit prior to creating PR
Were there any issues that arose?
-Nope!
Is there anything needed from your teammates?
-Timer view still needs a lot of work in CSS and the 'Start' button needs functionality built.
Next intended features
-Update data model when form is submitted.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1 +1,21 @@
...
+  event.preventDefault();
+  formView.classList.add("hidden");
+  timerView.classList.remove("hidden");
+  var categoryChoice = document.querySelector('input[name="activity_categories"]:checked');
+  if (categoryChoice.value === "study") {
+    categoryName.innerText = "Study";
+  } if (categoryChoice.value === "meditate") {
+    categoryName.innerText = "Meditate";
+  } if (categoryChoice.value === "exercise")
+    categoryName.innerText = "Exercise";
```
Comment (pthan1) [inline main.js:20]: I think we're missing the {} around the code block here.

Comment (pthan1) [inline main.js:26]: I think we're missing the {} around the code block here.

File: main.js [hunk 2]
```diff
@@ -1 +1,21 @@
...
+var activityInput = document.querySelector(".text-field-long");
+var timeInput = document.querySelector(".time-inputs-sections")
+var categoryName = document.querySelector(".category-name");
 
+submitForm.addEventListener("click", submitActivity);
+
+function submitActivity(event) {
+  event.preventDefault();
+  formView.classList.add("hidden");
+  timerView.classList.remove("hidden");
```
Comment (pthan1) [inline main.js]: Opportunity for a dynamic function?

File: main.js [hunk 3]
```diff
@@ -1 +1,21 @@
+var submitForm = document.querySelector(".activity-button");
+var formView = document.querySelector(".left-section");
+var timerView = document.querySelector(".timer-container");
+var activityInput = document.querySelector(".text-field-long");
+var timeInput = document.querySelector(".time-inputs-sections")
+var categoryName = document.querySelector(".category-name");
 
+submitForm.addEventListener("click", submitActivity);
+
+function submitActivity(event) {
```
Comment (pthan1) [inline main.js:16]: We should also add code to change the "New Activity" header to "Current Activity" per the project spec.

Comment (natesheridan) [reply main.js:16]: +1 
add on line 6.5:
`var leftSectionHeader = document.querySelector("#left-section-header")`

File: main.js (hunk 2 above)
Comment (natesheridan) [reply main.js]: replace 12-13 with :
```
function switchLeftDisplay(){
   timerView.classList.toggle("hidden");
   formView.classList.toggle("hidden");
}
```

File: main.js (hunk 1 above)
Comment (annnuuuh) [reply main.js:26]: good catch!

File: main.js (hunk 2 above)
Comment (annnuuuh) [reply main.js]: Good idea!

File: main.js (hunk 3 above)
Comment (annnuuuh) [reply main.js:16]: added that line to 6.5. Didn't change html because I have a feeling you probably did for the timer page, Nate! :)

**Review by natesheridan — APPROVED** [review]
Looks great!

**Pull request merged**